### DISCUSSION TITLE
Scale bar in bigwig tracks?

### DISCUSSION THREAD

**Discussion by srobb1** [GMOD/jbrowse#1654] [General]
Hi, I probably should know this, but what exactly is represented by the scale bar in the bigwig tracks. Is it reads per base?

![Screenshot 2024-11-19 at 3 50 10 PM](https://github.com/user-attachments/assets/e38a4f8f-3875-474b-8c11-899762586b90)

Thanks,
Sofia

**Comment by cmdcolin** [comment]
bigwigs represent data at a variety of "zoom levels". at the higher zoom levels (e.g. if you're viewing the whole chromosome) then you see large bin sizes, and the thing that jbrowse is displaying is the mean value (e.g. read depth) in that bin. 

if you zoomed in to the base level, the fractions might go away, because it will no longer be "binning" the values at all, and show you the raw values used to create the bigwig

there are a couple flags regarding this added in jbrowse 1 config here called "scoreType" https://jbrowse.org/docs/bigwig.html

i'd say the options there are a bit more confusing than they should be. jbrowse 1 additionally tries to do things related to average values if multiple values would plot at the same pixel. that part is a bit much.

jbrowse 2 takes a different approach where it

a) just "overplots" the data if it would plot at the same pixel. it rarely is the exact same pixel, but a fraction of a pixel, and is often visible anyways, and the averaging at pixel values introduces weird interpretability trouble
b) plots "whiskers": when you are zoomed out e.g. viewing whole chromosomes, it actually plots all three values: min, max, and mean, in slightly different shades. this helps you see big dips or amplifications that are otherwise washed out by mean values. the term "whiskers" for this is a UCSC-ism, they also (can) do this for their bigwig files.
c) and, if you don't want to view the whiskers in jbrowse 2,  you can change it in the UI

**Reply by cmdcolin** [reply]
example screenshot in jbrowse 2 showing how little "small peaks" are not visible when you just show the mean/average value (which is what jbrowse 1 shows by default) but is visible in the whiskers or max value only modes

![untitled(3)](https://github.com/user-attachments/assets/cc230c1a-2c1c-4f23-a044-61d32000af42)

**Comment by srobb1** [comment]
Thanks Colin!
